src/Calculations/FirstQuartile.py

```diff
@@ -1,13 +1,13 @@
-from src.Calculations import Calculation
+from src.Calculations.Calculation import Calculation, number
 import numpy
 
 
-class FirstQuartile(Calculation.Calculation):
+class FirstQuartile(Calculation):
     def __init__(self):
         super().__init__("First Quartile", vertical=True, horizontal=False)
 
     def do(self, data: [list]) -> list:
-        filtered_data = self.filter_out(data, str)
+        filtered_data = self.filter(data, number)
```

Comment:
yeah – there is a natural order for how this is implemented depending on who merges first. Someone will have to rebase and adjust their code after the rebase (whether it be me or you, it doesn't matter). so don't make any changes based on this now, wait until code is merged to main and always base your code off of main. if you change it now technically your code is then based off of this branch rather than main which is unnecessary